src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import org.hibernate.SessionFactory;
+
+public class ActorServiceImpl extends AbstractServiceImpl implements ActorService {
+    private final ActorDao actorDao = new ActorDaoImpl(factory);
+
+    public ActorServiceImpl(SessionFactory sessionFactory) {
+        super(sessionFactory);
+    }
```

Comment:
```suggestion
    private final ActorDao actorDao;

    public ActorServiceImpl(ActorDao actorDao) {
        this.actorDao = actorDao;
    }
```
the same for other services